docs/integrations/builtin/core-nodes/n8n-nodes-base.httprequest/index.md

```diff
@@ -0,0 +1,254 @@
+///
+??? Details "Understand pagination"
+    Pagination means splitting a large set of data into multiple pages. The amount of data on each page depends on the limit you set.
+  
+    For example, you make an API call to an endpoint called `/users`. The API wants to send back information on 300 users, but this is too much data for the API to send in one response. 
+  
+    If the API supports pagination, you can incrementally fetch the data. To do this, you call `/users` with a pagination limit, and a page number or URL to tell the API which page to send. In this example, say you use a limit of 10, and start from page 0. The API sends the first 10 users in its response. You then call the API again, increasing the page number by 1, to get the next 10 results.
+
+Configure the pagination settings:
+
+* **Pagination Mode**:
+    * **Off**: Turn off pagination.
+    * **Update a Parameter in Each Request**: Use this when you need to dynamically set parameters for each request.
+    * **Response Contains Next URL**: Use this when the API response includes the URL of the next page. Use an expression to set **Next URL**.
+
+For example setups, refer to [HTTP Request node cookbook | Pagination](/code/cookbook/http-node/pagination/).
+
+n8n provides built-in variables for working with HTTP node requests and responses when using pagination:
+
+--8<-- "_snippets/integrations/builtin/core-nodes/http/pagination-variables.md"
+
+--8<-- "_snippets/integrations/builtin/core-nodes/http/pagination-api-differences.md"
+
+### Proxy
+
+Use this option if you need to specify an HTTP proxy.
+
+Enter the **Proxy** the request should use.
+
+### Timeout
+
+Use this option to set how long the node should wait for the server to send response headers (and start the response body). The node aborts requests that exceed this value for the initial response.
+
+Enter the **Timeout** time to wait in milliseconds.
+
+## Import curl command
+
+[curl](https://curl.se/){:target=_blank .external-link} is a command line tool and library for transferring data with URLs.
+
+You can use curl to call REST APIs. If the API documentation of the service you want to use provides curl examples, you can copy them out of the documentation and into n8n to configure the HTTP Request node.
+
+Import a curl command:
+
+1. From the HTTP Request node's **Parameters** tab, select **Import cURL**. The **Import cURL command** modal opens.
+2. Paste your curl command into the text box.
+3. Select **Import**. n8n loads the request configuration into the node fields. This overwrites any existing configuration.
+
+## Templates and examples
+
+<!-- see https://www.notion.so/n8n/Pull-in-templates-for-the-integrations-pages-37c716837b804d30a33b47475f6e3780 -->
+[[ templatesWidget(title, 'http-request') ]]
```

Comment:
This is likely due to changing the page title...I could have sworn we already confirmed this worked but maybe because this is an index page it's funky. I'll test a bit more.

Reply:
@imchairmanm good catch on this. It appears the macro's been partially broken since we rolled out the SEO changes that tweaked the heading, and neither Aya nor I caught that.

I've updated the macro to reference page.title instead of title, and that seems to resolve all issues with it, so I'm releasing that change. Thanks for catching it!